### PR TITLE
Feature/pierreh/64/add top level imports

### LINKED ISSUE #64: Add top-level exports
Will make it easier to use the SDK without reaching for deep-level exports which may be tricky to use depending on bundler/platform

### PR DESCRIPTION
**Pull request by phieronymus**
Closes #64 

As I am not used to this, please feel free to tell me if I do something wrong even if that mean re-doing the whole thing 🙂

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -3,7 +3,7 @@ version: 2
 defaults: &defaults
   working_directory: ~/repo
   docker:
-    - image: circleci/node:12
+    - image: circleci/node:14
```
Comment (phieronymus) [inline .circleci/config.yml:6]: This is unrelated but I am pushing for all of our modules to be built on node 14 🙂

**Comment by phieronymus** [comment]
This is for the coverage: https://github.com/univapay/univapay-node/issues/69. It was broken at some point 🙂

File: .circleci/config.yml [hunk 2]
```diff
@@ -74,11 +74,7 @@ jobs:
 
       - run:
           name: Publish package LIB
-          command: npm publish lib
-
-      - run:
-          name: Publish package ES
-          command: npm publish es
+          command: npm publish dist
```
Comment (perrin4869) [inline .circleci/config.yml:77]: this is just `npm publish`!

**Review by perrin4869 — COMMENTED** [review]
Previously we were building two separate packages under `lib` and `es`, but now we are publishing a single package at the top level, so it's just `npm publish` (without `dist`)

File: .circleci/config.yml (hunk 2 above)
Comment (phieronymus) [reply .circleci/config.yml]: Thanks. That makes sense 🙂 

Somehow I forgot to merge pull `master` first and looking at the `package.json` it was requiring the index 🤦 My brain was probably very slow yesterday.

**Review by laxa88 — APPROVED** [review]

**Review by perrin4869 — APPROVED** [review]

**Pull request merged**